src/content/release-notes.mdx

```diff
@@ -7,26 +7,28 @@ id: release-notes
 
 ## 1.29.0
 
-XX February 2025
+7 February 2025
 
 This version is compatible with Kubernetes versions 1.28 to 1.31 \
 Okteto Chart release 1.29 is designed to work with [Okteto CLI 3.4.x](https://github.com/okteto/okteto/releases/tag/3.4.0)
 
-### Breaking Changes {#breaking-changes-1.29}
-
-TBC
 
 ### New Features {#new-features-1.29}
 
-TBC
+- Added support for [Kubernetes 1.31](https://kubernetes.io/blog/2024/08/13/kubernetes-v1-31-release/)  <!-- 9081 -->
+- Personal Namespaces can now be [included in your Garbage Collection Policy](admin/cleanup.mdx#applying-garbage-collection-to-personal-namespaces) <!-- 9056 -->
+- Okta users can now configure automatic deprovisioning of user accounts through Okteto  <!-- 9064 -->
 
 ### Improvements {#improvements-1.29}
 
-TBC
+- Improved the retry mechanism in `okteto up` to handle overnight disconnections more gracefully by treating syncthing timeouts as transient after a successful initial execution  <!-- 9069 -->
```

Comment:
I wouldn't mention this for now. This was something we did specifically for one customer and it is behind an unsupported feature flag. We should discuss when to make GA and document it properly then